server/core/src/irods_report_plugins_in_json.cpp

```diff
@@ -97,6 +97,13 @@ namespace irods {
             return PASS(ret);
         }
 
+        rodsLong_t id;
+        ret = _resc->get_property< rodsLong_t >( irods::RESOURCE_ID, id );
+        if ( !ret.ok() ) {
+            return PASS(ret);
+        }
+        std::string id_str = std::to_string( id );
```

Comment:
Could the `std::to_string` call occur on 156 at the time of insertion into the `_entry` map?

Reply:
Sure thing